bindings/python/tests/geosciences/test-py-cross-section.py

```diff
@@ -189,6 +189,51 @@ def test_io( model ):
     reloaded_model = geosciences.load_cross_section( "test.og_xsctn" )
     check_reloaded_model( reloaded_model )
 
+def test_copy(model):
+    copy = geosciences.CrossSection()
+    copier = geosciences.CrossSectionBuilder( copy )
+    copier.copy( model )
+
+    if copy.nb_surfaces() != 4:
+        raise ValueError("[Test] Number of surfaces in copied model should be 4" )
+    if copy.nb_lines() != 8:
+        raise ValueError("[Test] Number of lines in copied model should be 8" )
+    if copy.nb_horizons() != 3:
+        raise ValueError("[Test] Number of horizons in copied model should be 3" )
+    if copy.nb_faults() != 2:
+        raise ValueError("[Test] Number of faults in copied model should be 2" )
+    if copy.nb_fault_blocks() != 2:
+        raise ValueError("[Test] Number of fault blocks in copied model should be 2" )
+    if copy.nb_stratigraphic_units() != 2:
+        raise ValueError("[Test] Number of stratigraphic units in copied model should be 2" )
+   # nb_surface_boundaries = 0 
```

Comment:
Here! This code is not working but it should as it works in C++. It seems that copy.boundary(surface) does not work in this case.